client/valor/schemas/compatibility.py

```diff
@@ -0,0 +1,211 @@
+from typing import Any, Optional, Union
```

Comment:
is there a more descriptive name than "compatibility"? perhaps "utils"?

Reply:
This file will be removed in my next PR, it only exists to maintain compatibility between now and then.

Reply:
I don't love that approach given I don't know what will be included in the next PR, but I'm willing to live with it